yt/yt/client/api/operation_client.h

```diff
@@ -361,6 +364,20 @@ struct TListJobsResult
     std::vector<TError> Errors;
 };
 
+struct TPagedLogReq
+{
+    i64 Limit = 0;
+    i64 Offset = 0;
+};
+
+struct TPagedLog
+{
+    TString Data;
```

Comment:
I see a lost of casts from TString to TSharedRef and backwards. Maybe it's better to make this field `TSharedRef`?